Casks/p/proton-pass.rb

```diff
@@ -0,0 +1,28 @@
+cask "proton-pass" do
+  version "1.17.5"
+  sha256 "2331a0974c047c0115b9ad76db16ca5d0a4b65987bd84b2bdcb1aeef1c9d198b"
+
+  url "https://proton.me/download/PassDesktop/darwin/universal/ProtonPass_#{version}.dmg"
+  name "Proton Pass"
+  desc "Desktop client for Proton Pass"
+  homepage "https://proton.me/pass"
+
+  livecheck do
+    # TODO: add when Proton Pass releases resource
+    skip "No version information available"
+  end
```

Comment:
```suggestion
  livecheck do
    url "https://proton.me/download/PassDesktop/darwin/universal/version.json"
    strategy :json do |json|
      json["Releases"].select { |item| item["CategoryName"]&.match?("Stable") }
                      .map { |item| item["Version"] }
    end
  end
```

Proton Pass seems to be an Electron application, but I can't find an `app-update.yml` in the app directory (I didn't find a button to trigger check update manually). This url is found on the download page.